custom_components/octopus_intelligent/__init__.py

```diff
@@ -46,7 +46,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     )
 
     try:
-        await hass.async_add_executor_job(octopus_system.start)
+        await octopus_system.start()
```

Comment:
This change is a git cherry-pick from PR #38 because this PR depends on that one.